service/create/service.go

```diff
@@ -567,7 +567,23 @@ func (s *Service) Boot() {
 				},
 				DeleteFunc: func(obj interface{}) {
 					// TODO(nhlfr): Move this to a separate operator.
-					cluster := *obj.(*awstpr.CustomObject)
+					var cluster awstpr.CustomObject
```

Comment:
Would be cool to have comments based on the PR description and definition of the `DeletedFinalStateUnknown` event as Lorenzo posted it.